owa/app/js/subscription/subscription.controller.js

```diff
@@ -48,6 +51,27 @@ class SubscriptionController {
         vm.subscription = response.results[0];
       })
     }
+
+    /*
+    *This checks if collection version has been passed to subscription url by checking number of forward slashes
+    *If the number is 5, such as with https://api.openconceptlab.org/users/username/collections/collectionname/v1.0
```

Comment:
Is the number 5 or 7?

Reply:
I already documented this

Reply:
What i mean is that the number is not 5 in the example you have given.

Reply:
I have update the descriptions